imap_processing/swe/l1a/swe_l1a.py

```diff
@@ -13,15 +93,23 @@ def swe_l1a(packet_file: str):
 
     Parameters
     ----------
-    packet_file : str
-        The path and filename to the L0 file to read
+    packets: list
+        Decom data list that contains all appIds
     """
-    decom_data = decom_swe.decom_packets(packet_file)
-    logging.info(f"Unpacking data from {packet_file}")
-
-    # If appId is science, then the file should contain all data of science appId
-    if decom_data[0].header["PKT_APID"].raw_value == 1344:
-        logging.info("Processing science data")
-        return swe_science(decom_data=decom_data)
-    else:
-        return decom_data
+    # group data by appId
+    grouped_data = group_by_apid(packets)
+
+    for apid in grouped_data.keys():
+        # If appId is science, then the file should contain all data of science appId
+        if apid == SWEAPID.SWE_SCIENCE.value:
+            # sort data by acquisition time
+            sorted_packets = sorted(
+                grouped_data[apid], key=lambda x: x.data["ACQ_START_COARSE"].raw_value
+            )
+            logging.info("Processing science data")
```

Comment:
```suggestion
            logging.debug("Processing science data for [%s] packets", len(sorted_packets))
```
Add some more information to the log to help debug later? Also make it a debug log since we don't need it every time and likely only when we are trying to investigate something.